### ISSUE TITLE
Update to 3.5.1 not working

### ISSUE THREAD

**Issue by alpharesearch** [sdesalve/hassio-addons#32]
I tried it a few times. Here is the info from the log file:

21-01-21 23:21:14 INFO (SyncWorker_5) [supervisor.docker.interface] Updating image 89275b70/armv7-addon-dss_voip:3.4.3 to 89275b70/armv7-addon-dss_voip:3.5.1
21-01-21 23:21:14 INFO (SyncWorker_5) [supervisor.docker.addon] Starting build for 89275b70/armv7-addon-dss_voip:3.5.1
21-01-21 23:21:17 ERROR (SyncWorker_5) [supervisor.docker.addon] Can't build 89275b70/armv7-addon-dss_voip:3.5.1: The command '/bin/ash -o pipefail -c apk add --no-cache     coreutils=8.32-r2     wget=1.21.1-r1     sox     pjsua     screen     bc' returned a non-zero code: 6
21-01-21 23:21:17 ERROR (SyncWorker_5) [supervisor.docker.addon] Build log: 
Step 1/20 : ARG BUILD_FROM=ghcr.io/hassio-addons/base/amd64:9.0.0

Step 2/20 : FROM ${BUILD_FROM}

 ---> 8b4f1febe3a8

Step 3/20 : ENV     S6_BEHAVIOUR_IF_STAGE2_FAILS=2     S6_CMD_WAIT_FOR_SERVICES=1     TERM="xterm-256color"

 ---> Using cache

 ---> e09cce5ad062

Step 4/20 : ENV LANG C.UTF-8

 ---> Using cache

 ---> 61f4a734af17

Step 5/20 : COPY rootfs /

 ---> Using cache

 ---> 3fccd76e475b

Step 6/20 : RUN apk add --no-cache     coreutils=8.32-r2     wget=1.21.1-r1     sox     pjsua     screen     bc

 ---> Running in 7452e84f0d0d

fetch https://dl-cdn.alpinelinux.org/alpine/v3.13/main/armv7/APKINDEX.tar.gz
fetch https://dl-cdn.alpinelinux.org/alpine/v3.13/community/armv7/APKINDEX.tar.gz

WARNING: Ignoring https://dl-cdn.alpinelinux.org/alpine/v3.13/main: temporary error (try again later)

WARNING: Ignoring https://dl-cdn.alpinelinux.org/alpine/v3.13/community: temporary error (try again later)

ERROR: unable to select packages:

  bc (no such package):
    required by: world[bc]
  coreutils (no such package):
    required by: world[coreutils=8.32-r2]
  pjsua (no such package):
    required by: world[pjsua]
  screen (no such package):
    required by: world[screen]
  sox (no such package):
    required by: world[sox]
  wget (no such package):
    required by: world[wget=1.21.1-r1]

Removing intermediate container 7452e84f0d0d

**Comment by sdesalve** [comment]
your host machine? OS?

**Comment by sdesalve** [comment]
please try to upldate to 3.5.3

**Comment by alpharesearch** [comment]
I just tried the latest version and still the same issue. 

I'm on a Raspberry Pi 4 with 4 GB ram and 512GB SSD on Raspbian OS. I guess on part of the issue is that amd64 stuff:

[supervisor.docker.addon] Build log: 
Step 1/20 : ARG BUILD_FROM=ghcr.io/hassio-addons/base/amd64:9.0.1

**Comment by sdesalve** [comment]
It can be...

Could you try to install this addon?

https://github.com/hassio-addons/addon-example

It's example addon updated this week

**Comment by alpharesearch** [comment]
Installing the example worked. Armv7 vs amd64, not sure why it's not using the right architecture. 

21-01-22 12:40:14 INFO (MainThread) [supervisor.addons] Creating Home Assistant add-on data folder /data/addons/data/a0d7b954_example
21-01-22 12:40:14 INFO (SyncWorker_2) [supervisor.docker.interface] Downloading docker image ghcr.io/hassio-addons/example/armv7 with tag 4.0.4.
21-01-22 12:40:17 INFO (MainThread) [supervisor.addons] Add-on 'a0d7b954_example' successfully installed

**Comment by sdesalve** [comment]
I don't this it can be a base image error. 

On first log you have
21-01-21 23:21:14 INFO (SyncWorker_5) [supervisor.docker.interface] Updating image 89275b70/**armv7**-addon-dss_voip:3.4.3 to 89275b70/armv7-addon-dss_voip:3.5.1

Please try
- connect to SSH addon and execute this sequence of commands. Error are related to apk add command 
```
apk add --no-cache coreutils=8.32-r2 
apk add --no-cache wget=1.21.1-r1 
apk add --no-cache sox 
apk add --no-cache pjsua 
apk add --no-cache screen 
apk add --no-cache bc
```

You can try also to copy addon configuration, uninstall it and try to reinstall (make an Hassio snapshot BEFORE). Other users on Raspberry + HassOS have upgraded without any error... 
Do you have latest Hassio + Supervisor? RaspbianOS is supported by Hassio?

**Comment by alpharesearch** [comment]
System: Raspbian GNU/Linux 10 (buster) (armv7 / raspberrypi4)
Home Assistant Core: 2021.1.4
Home Assistant Supervisor: 2021.01.5

I believe I have everything up to date, but only Debian is supported - now Raspbian is pretty much debian

I'm executing this on the "Terminal & SSH" V8.10.0 add on (looks like I have newer packages?):
~ $ apk add --no-cache coreutils=8.32-r2 
fetch http://dl-cdn.alpinelinux.org/alpine/v3.12/main/armv7/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.12/community/armv7/APKINDEX.tar.gz
**ERROR: unsatisfiable constraints:
  coreutils-8.32-r0:
    breaks: world[coreutils=8.32-r2]**
~ $ apk add --no-cache wget=1.21.1-r1 
fetch http://dl-cdn.alpinelinux.org/alpine/v3.12/main/armv7/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.12/community/armv7/APKINDEX.tar.gz
**ERROR: unsatisfiable constraints:
  wget-1.20.3-r1:
    breaks: world[wget=1.21.1-r1]**
~ $ apk add --no-cache sox 
fetch http://dl-cdn.alpinelinux.org/alpine/v3.12/main/armv7/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.12/community/armv7/APKINDEX.tar.gz
(1/9) Installing libao (1.2.2-r0)
(2/9) Installing gsm (1.0.19-r0)
(3/9) Installing libid3tag (0.15.1b-r7)
(4/9) Installing libmad (0.15.1b-r9)
(5/9) Installing lame (3.100-r0)
(6/9) Installing opus (1.3.1-r0)
(7/9) Installing opusfile (0.11-r2)
(8/9) Installing libpng (1.6.37-r1)
(9/9) Installing sox (14.4.2-r5)
Executing busybox-1.31.1-r19.trigger
OK: 87 MiB in 104 packages
~ $ apk add --no-cache pjsua 
fetch http://dl-cdn.alpinelinux.org/alpine/v3.12/main/armv7/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.12/community/armv7/APKINDEX.tar.gz
(1/7) Installing libsamplerate (0.1.9-r1)
(2/7) Installing jack (1.9.14-r0)
(3/7) Installing portaudio (190600.20161030-r1)
(4/7) Installing speex (1.2.0-r0)
(5/7) Installing libsrtp (2.3.0-r0)
(6/7) Installing pjproject (2.9-r0)
(7/7) Installing pjsua (2.9-r0)
Executing busybox-1.31.1-r19.trigger
OK: 91 MiB in 111 packages
~ $ apk add --no-cache screen 
fetch http://dl-cdn.alpinelinux.org/alpine/v3.12/main/armv7/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.12/community/armv7/APKINDEX.tar.gz
(1/1) Installing screen (4.8.0-r0)
Executing busybox-1.31.1-r19.trigger
OK: 92 MiB in 112 packages
~ $ apk add --no-cache bc
fetch http://dl-cdn.alpinelinux.org/alpine/v3.12/main/armv7/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.12/community/armv7/APKINDEX.tar.gz
(1/1) Installing bc (1.07.1-r1)
Executing busybox-1.31.1-r19.trigger
OK: 92 MiB in 113 packages
~ $

**Comment by sdesalve** [comment]
Sorry, try

apk add --no-cache coreutils 
apk add --no-cache wget

**Comment by alpharesearch** [comment]
I did try to reboot the host, but that didn't change anything...

When I connect a bash via Portainer to the example container I can install all the packages (but some are already installed) and apk list shows that everything is installed with the right version too.

root@a0d7b954-example:/$ apk list
WARNING: Ignoring https://dl-cdn.alpinelinux.org/alpine/v3.13/main: No such file or directory
WARNING: Ignoring https://dl-cdn.alpinelinux.org/alpine/v3.13/community: No such file or directory
pjproject-2.9-r0 armv7 {pjproject} (GPL-2.0-or-later) [installed]
wget-1.21.1-r1 armv7 {wget} (GPL-3.0-or-later WITH OpenSSL-Exception) [installed]
portaudio-190600.20161030-r1 armv7 {portaudio} (MIT) [installed]
coreutils-8.32-r2 armv7 {coreutils} (GPL-3.0-or-later) [installed]
...

Btw, the commands without explicit versions worked in the SSH add on too, but installs the older versions...

~ $ apk add --no-cache coreutils
fetch http://dl-cdn.alpinelinux.org/alpine/v3.12/main/armv7/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.12/community/armv7/APKINDEX.tar.gz
(1/3) Installing libacl (2.2.53-r0)
(2/3) Installing libattr (2.4.48-r0)
(3/3) Installing coreutils (8.32-r0)
Executing busybox-1.31.1-r19.trigger
OK: 86 MiB in 98 packages
~ $ apk add --no-cache wget
fetch http://dl-cdn.alpinelinux.org/alpine/v3.12/main/armv7/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.12/community/armv7/APKINDEX.tar.gz
(1/3) Installing libunistring (0.9.10-r0)
(2/3) Installing libidn2 (2.3.0-r0)
(3/3) Installing wget (1.20.3-r1)
Executing busybox-1.31.1-r19.trigger
OK: 88 MiB in 101 packages
~ $

**Comment by alpharesearch** [comment]
I did a test and something very strange is happening that shows there are some things not right, but it has nothing to do with your repository. 
I can install hassio-addons/addon-example from the default repository just fine. 

BUT when I copy the addon-example from github to my addon repository and try to install I get the exact same issue like with your repository. 

I'm using Linux Mint and I did clone both to my local PC and just copied the folder over via cp and git push. All the new files are online and look the same like in the original. ???
